README.md

```diff
@@ -29,7 +29,7 @@ Or manually:
 
 
 An example of using the Ant task inside Pipeline DSL.
-This code snippet must be put inside a `step` block when using the declarative syntax.
+This code snippet must be put inside a `script` block when using the declarative syntax.
```

Comment:
```suggestion
This code snippet must be put inside a `steps` block when using the Declarative syntax.
```

See https://www.jenkins.io/doc/book/pipeline/syntax/#steps.

Reply:
I get an error when I try to use this declarative pipeline that includes the `withAnt` block in `steps` without `script`:

```
pipeline {
    agent any

    stages {
        stage('Hello') {
            steps {
                withAnt(installation: 'ant') {
                    dir("scoring") {
                        if (isUnix()) {
                            sh "ant mytarget"
                        } else {
                            bat "ant mytarget"
                        }
                    }
                }
            }
        }
    }
}
```

The error is:

```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
07:41:21  WorkflowScript: 9: Expected a step @ line 9, column 25.
07:41:21                             if (isUnix()) {
07:41:21                             ^
07:41:21  
07:41:21  1 error
```

I could remove the `isUnix` conditional from the example, but I assumed that was an intentional choice, since the Pipeline needs to choose whether to use `sh` or `bat` or `powershell` to invoke the `ant` command.

Reply:
Oh because of the `if` switch, I missed that. Otherwise nested steps are supported within `steps` (AFAIK).